dms/views/assets.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- License AGPL-3.0 (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <template id="dms_frontend_demo" inherit_id="web.assets_frontend">
```

Comment:
Please include this as demo data, as there's not too much sense to load this for production deployments.